### PR TITLE
Fix/3274 make the password field fixed on the screen

### PR DESCRIPTION
**Pull request by gbonaspetti**
#3274

## Checkpoints

The goal of this section is to help code reviewers and the merge request author to do the required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [x] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [x] Tested by the quality team

### CODE REVIEW

File: frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        wrapper.setState({ newUserUsername: '' })
+        expect(wrapper.instance().isValidateButtonDisabled()).to.equal(true)
+      })
+
+      it('should return true if the name or the profile are empty', () => {
+        wrapper.setProps({ emailNotifActivated: false })
+        expect(wrapper.instance().isValidateButtonDisabled()).to.equal(true)
+      })
+
+      it('should return true if the email notification are disactivated and the password are empty', () => {
```
Comment (Skylsmoi) [inline frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js:31]: disactivated doesn't exist in English, the word is `deactivated`

File: frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      it('should return true if the email notification are activated, the email and the password are empty', () => {
+        wrapper.setState({ newUserEmail: '' })
+        wrapper.setState({ newUserPassword: '' })
+        wrapper.setState({ newUserName: '' })
+        wrapper.setState({ newUserProfile: '' })
+        wrapper.setState({ newUserUsername: '' })
+        expect(wrapper.instance().isValidateButtonDisabled()).to.equal(true)
+      })
+
+      it('should return true if the name or the profile are empty', () => {
```
Comment (Skylsmoi) [inline frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js:26]: it seems that this test expect empty values that are set in the previous test.
This make it looks like tests aren't independent.
If I'm correct, you should put some `wrapper.setState` of the previous test in a `before()` function

File: frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js (hunk 1 above)
Comment (gbonaspetti) [reply frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js:31]: changed

File: frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js (hunk 2 above)
Comment (gbonaspetti) [reply frontend_app_admin_workspace_user/test/component/AddUserForm.spec.js:32]: changed

**Pull request merged**